### PR TITLE
Make limitations of PT-based demux more clear, based on Magnus's feed…

### PR DESCRIPTION
**Pull request by pthatcherg**
This is based on the feedback from Magnus.

### CODE REVIEW

**Comment by aboba** [comment]
SSRCs can move between m-lines in some situations, right?

If the application calls transceiver.stop(), a BYE will be sent for the SSRC and after negotiation, the m-line will have a zero port number. If a new O/A exchange were to put that former-SSRC on a new m-line, it will be signaled and installed into the SSRC table without issue, correct?

**Comment by pthatcherg** [comment]
True.  I guess we'd have to clarify that by it can't move by simply sending RTP packets with the PT tied to another m= section.  Some kind of reconfiguration/renegotiation would be needed.

**Comment by fluffy** [comment]
I don't really care if we add something this but this seems 100% clear from the existing text so I really don't see the point of adding it.

**Comment by pthatcherg** [comment]
Bernard, I made it more clear that there are other mechanisms for "moving" an SSRC, such as the one you mentioned.  I also made the text more explicit.  I agree with Cullen that this isn't really needed, since it's clear from the algorithm.  But Magnus asked for it be extra super explicit, so I'm willing to add it.

**Comment by gloinul** [comment]
Thanks, I would probably not have added as much. But, I think there is a point to make clear the limitations. I am not that happy about the added text in response to Bernard's comment. Yes, this could occur, but a re-added stream after having called stop (BYE) is a new RTP stream, and thus a new random SSRC, even if signalled explicit. I don't want to encourage behaviors with potential downsides. I think using MID is the obvious alternative that should be noted if anything. And that goes back to my initial motivation for this text, that MID is the better alternative.

**Comment by pthatcherg** [comment]
Magnus, I made the text much shorter.  I also removed the "example" that would encourage people in the wrong direction.

**Comment by gloinul** [comment]
This looks good to me.

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -6602,8 +6602,14 @@ a=msid:bbce3ba6-abfc-ac63-d00a-e15b286f8fce
       when using BUNDLE, and instead use a payload type based mechanism
       for demuxing. In this situation, each "m=" line MUST use unique
       payload type values, in order for the payload type to be a
-      reliable indicator of the relevant "m=" line for the RTP
-      stream.</t>
+      reliable indicator of the relevant "m=" line for the RTP stream.
+      Note that when using payload type based demuxing, an SSRC
+      will be mapped to an "m=" line by first packet with that
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: "the first packet"

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -6602,8 +6602,14 @@ a=msid:bbce3ba6-abfc-ac63-d00a-e15b286f8fce
...
+      SSRC and the mapping will not be changed even if the same
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: comma after 'SSRC'

**Comment by fluffy** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (pthatcherg) [inline draft-ietf-rtcweb-jsep.xml]: fixed

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (pthatcherg) [inline draft-ietf-rtcweb-jsep.xml]: fixed

**Comment by pthatcherg** [comment]
FYI, I updated https://github.com/cdh4u/draft-sdp-bundle/pull/29 to match.

**Pull request merged**